ra2ce/network/network_config_data/network_config_data.py

```diff
@@ -78,9 +78,10 @@ class IsolationSection:
 class HazardSection:
     hazard_map: list[Path] = field(default_factory=list)
     hazard_id: str = ""
-    hazard_field_name: list[str] = field(default_factory=list)
```

Comment:
Why has this been renamed? For safety I would leave it as it was (if it works don't touch it). When we were doing the mapping `.ini` to `object` we actually discussed several of this options with users / developers. So although it's not entirely clear now I will not modify it and leave for a different issue.

Reply:
I changed it based on the downstream code.
Asking the team didn't result in a conclusive answer, so I will revert.
https://teams.microsoft.com/l/message/19:284af58b001c4613b667b3d7f35af30b@thread.tacv2/1712226038653?tenantId=15f3fe0e-d712-4981-bc7c-fe949af215bb&groupId=b3107d8a-8626-4b5d-b75c-4c5aeaa02a38&parentMessageId=1712226038653&teamName=Team%20GFS%20and%20friends&channelName=RA2CE&createdTime=1712226038653